solutions/devsprint-ana-nogal-1/DeliveryApp/Screens/Home/HomeViewController.swift

```diff
@@ -17,20 +21,27 @@ class HomeViewController: UIViewController {
         return homeView
     }()
 
+    lazy var searchController: UISearchController = {
+       let searchController = UISearchController(searchResultsController: nil)
+        searchController.searchResultsUpdater = self
```

Comment:
hummm 🤔 .... Aqui não deveríamos usar antes:
```suggestion
        searchController.searchBar.delegate = self
```

Reply:
corrigido.